internal/model/labels.go

```diff
@@ -83,6 +118,8 @@ type NumericLabelValue struct {
 	Values []float64
 	// Value holds the label `float64` value.
 	Value float64
+	// Global is `true` when the label has been decoded from the metadata.
+	Global bool
 }
 
 func (l NumericLabels) Set(k string, v float64) {
```

Comment:
```suggestion
// Set sets the label k to value v. If there existed a label in l with the same key,
// it will be replaced and its Global field will be set to false.
func (l NumericLabels) Set(k string, v float64) {
```

"Metadata" is kind of specific to the Elastic APM protocol, so I'd like to avoid using that term in the model, which may come from OpenTelemetry.